src/test/java/org/jenkinsci/plugins/durabletask/WindowsBatchScriptTest.java

```diff
@@ -114,6 +114,22 @@ private void testWithPath(String path) throws Exception {
         c.cleanup(ws);
     }
 
+    @Test public void exitCommandUnsignedInt() throws Exception {
+        Controller c = new WindowsBatchScript("echo hello world\r\nexit 3221225477").launch(new EnvVars(), ws, launcher, listener);
+        awaitCompletion(c);
+        ByteArrayOutputStream baos = new ByteArrayOutputStream();
+        c.writeLog(ws, baos);
+        if (enableBinary) {
+            assertEquals(Integer.valueOf(Integer.MAX_VALUE), c.exitStatus(ws, launcher, listener));
+        } else {
+            assertEquals(Integer.valueOf(-1073741819), c.exitStatus(ws, launcher, listener));
+        }
```

Comment:
Is this the difference between https://github.com/jenkinsci/lib-durable-task/blob/c823f6cb78f5208589b41e5ba467635b5ea121c1/src/pkg/common/common.go#L151-L157 vs. https://github.com/jenkinsci/durable-task-plugin/blob/cc2bd5774993fb6d30af1c28a91178ea463cb646/src/main/java/org/jenkinsci/plugins/durabletask/WindowsBatchScript.java#L148 ? Not great there is this inconsistency, though I suppose it is unlikely to matter much in practice (either way it is a failure).

Reply:
Yikes, so windows is always using unsigned int to return error codes? If so, I think lib-durable needs to be updated